docs/content/docs/install/uninstallation_guide.md

```diff
@@ -8,44 +8,136 @@ weight: 3
+
+For example, if [Hashicorp Vault](https://github.com/openservicemesh/osm/blob/e1c48b9b1d67be2f5b7ec90af4572fa6d5d42fd2/docs/content/docs/tasks_usage/certificates.md#installing-hashi-vault) was deployed for the sole purpose of managing certificates for OSM, all related resources can be deleted.
+
+### Delete OSM Namespace
+
+When installing a mesh, the `osm` CLI creates the namespace the control plane is installed into if it does not already exist. However, when uninstalling the same mesh, the namespace it lives in does not automatically get deleted by the `osm` CLI. This behavior occurs because
+there may be resources a user created in the namespace that they may not want automatically deleted.
+
+If the namespace was only used for OSM and there is nothing that needs to be kept around, the namespace can be deleted at this time with `kubectl`.
+
+```console
+$ kubectl delete namespace <namespace>
+namespace "<namespace>" deleted
+```
+
+Repeat the steps above for each mesh installed in the cluster. After there are no OSM control planes remaining, move to following step.
+
+## Remove OSM Cluster Wide resources
+
+OSM ensures that there the Service Mesh Interface Custom Resource Definitions(CRDs) exist in the cluster at install time. If they are not already installed, the `osm` CLI will install them before installing the rest of the control plane components. This is the same behvior when using the Helm charts to install OSM as well. CRDs are cluster-wide resources and may be used by other instances of OSM in the same cluster
+or other service meshes running in the same cluster. If there are no other instances of OSM or other service meshes running in the same
+cluster, these CRDs and instances of the SMI custom resources can be removed from the cluster using `kubectl`. When the CRD is deleted, all
+instances of that CRD will also be deleted.
+
+Run the following `kubectl` commands:
 
-1. Existing Envoy sidecar containers
-   - Redeploy application pods to delete sidecars
-1. Envoy bootstrap config secrets (stored in the application namespace)
-1. Namespace annotations, including but not limited to `openservicemesh.io/monitored-by`
-1. Custom resource definitions ([CRDs](https://github.com/openservicemesh/osm/tree/main/charts/osm/crds))
-1. Vault resources provided by the user
-1. The namespace in which OSM was installed
+kubectl delete -f [https://raw.githubusercontent.com/openservicemesh/osm/v0.8.0/charts/osm/crds/access.yaml](https://raw.githubusercontent.com/openservicemesh/osm/v0.8.0/charts/osm/crds/access.yaml)
+kubectl delete -f [https://raw.githubusercontent.com/openservicemesh/osm/v0.8.0/charts/osm/crds/specs.yaml](https://raw.githubusercontent.com/openservicemesh/osm/v0.8.0/charts/osm/crds/specs.yaml)
+kubectl delete -f [https://raw.githubusercontent.com/openservicemesh/osm/v0.8.0/charts/osm/crds/split.yaml](https://raw.githubusercontent.com/openservicemesh/osm/v0.8.0/charts/osm/crds/split.yaml)
```

Comment:
Does it make sense to use the `release-v0.8` branch instead of the `v0.8.0` tag?
Like this:  https://raw.githubusercontent.com/openservicemesh/osm/release-v0.8/charts/osm/crds/access.yaml

In this https://github.com/openservicemesh/osm/pull/3007 I decided to use a link to the `release-v0.8` branch instead of the tag.  I'm happy to change this if you think the tag is better approach

```
this ---- https://raw.githubusercontent.com/openservicemesh/osm/v0.8.0/charts/osm/crds/access.yaml
vs this - https://raw.githubusercontent.com/openservicemesh/osm/release-v0.8/charts/osm/crds/access.yaml
```

Reply:
updated. thanks